documentation/Cohen Hubbard Oesterwinter.tex

```diff
@@ -0,0 +1,166 @@
+\documentclass[10pt, a4paper, twoside]{basestyle}
+
+\usepackage[Mathematics]{semtex}
+\usepackage{chngcntr}
+\counterwithout{equation}{section}
+
+%%%% Shorthands.
+
+%%%% Title and authors.
+
+\title{On a Formula by Cohen, Hubbard and Oesterwinter}
+\date{\printdate{2021-03-13}}
+\author{Robin~Leroy (eggrobin) \& Pascal~Leroy (phl)}
+\begin{document}
+\maketitle
+\begin{sloppypar}
+\noindent
+This document proves and generalizes a formula given in \cite{CohenHubbardOesterwinter1973} to compute the velocity of a body in the context of numerical integration of $n$-body systems.
+\end{sloppypar}
+
+\subsection*{Statement}
+\cite{CohenHubbardOesterwinter1973} states that
+\begin{quotation}
+the formula for a velocity component [is] of the form:
+\begin{equation}
+\TimeDerivative{x}_{n + 1} = \frac{1}{h}\pa{x_n - x_{n - 1} + h^2 \sum{i = 0}[12] \gb_i \SecondTimeDerivative{x}_{n-i}}
+\label{eqncho}
+\end{equation}
+\end{quotation}
+They then proceed to tabulate explicit values for the (rational) coefficients $\gb_i$ without explaining how they are computed.  This makes it impossible to use this formula for integrators of a different order or to construct similar formulæ for slightly different purposes.
+
+\subsection*{Finite difference formulæ}
+In this section we prove a backward difference formula that makes it possible to compute an approximation of any derivative of a function on an equally-spaced grid at any desired order.  We then derive a corollary based on the second derivative $f\dersecond\of{x}$ that is useful for the following sections. 
+\begin{lemma}
+Given a sufficient regular function $f\of{x}$ and two integers $m ≤ n$, there exists a family of rational numbers $\gl^{m}_{n,\gn}$, with $0 ≤ \gn ≤ n$, such that:
```

Comment:
Given <del>a sufficient regular function $f\of{x}$ and</del> two integers $m ≤ n$, there exists a family of rational numbers $\gl^{m}_{n,\gn}$, with $0 ≤ \gn ≤ n$, such that <ins>**for any** sufficient regular function $f<del>\of{x}$</ins>